packages/wpcom.js/package.json

```diff
@@ -0,0 +1,48 @@
+{
+	"name": "wpcom",
+	"description": "Official JavaScript library for the WordPress.com REST API",
+	"version": "5.4.2",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"author": "Automattic, Inc.",
+	"scripts": {
+		"clean": "npx rimraf dist",
+		"prepublish": "npm run clean",
+		"prepare": "transpile"
+	},
+	"contributors": [
+		"Damian Suarez <rdsuarez@gmail.com>",
+		"Dennis Snell <dennis.snell@automattic.com>",
+		"Nathan Rajlich <nathan@automattic.com>",
+		"Matt Sherman <matt.sherman@automattic.com>",
+		"Timmy Crawford <timmy.crawford@automattic.com>",
+		"Benoit Person <benoit.person@gmail.com>",
+		"Jake Fried <jake.fried@automattic.com>",
+		"Jarda Snajdr <jsnajdr@gmail.com>"
+	],
+	"keywords": [
+		"wordpress",
+		"wordpress.com",
+		"browser",
+		"rest",
+		"api",
+		"library"
+	],
+	"repository": "git@github.com:Automattic/wpcom.js.git",
+	"license": "MIT",
+	"files": [
+		"dist",
+		"History.md",
+		"README.md"
+	],
+	"browser": {
+		"./dist/cjs/lib/util/fs.js": "./dist/cjs/lib/util/fs-browser.js",
```

Comment:
I'm not sure if we need to do this in the `cjs` version, but it's probably safe to do so.

Reply:
I think we need it both in `cjs` and in `esm` versions. It makes sure that `import './fs'` and `require( './fs' )` are resolved to `fs-browser.js` when running in browser.